apps/builder/app/shared/instance-utils.ts

```diff
@@ -697,7 +697,8 @@ const traverseStyleValue = (
     value.type === "invalid" ||
     value.type === "unset" ||
     value.type === "rgb" ||
-    value.type === "function"
+    value.type === "function" ||
+    value.type === "guaranteedInvalid"
```

Comment:
What's the point of having this type? is it to allow an invalid value while not showing an error? This sounds like it could be called "initial"

Reply:
Guaranteed-invalid is the name of initial value of custom properties. "initial" should be always resolved into something. This value is what it resolves into. And it cannot be any other value because the logic use it to fallback into initial value of property which use custom property.